App.tsx

```diff
@@ -106,10 +108,11 @@ export const App: React.FC<AppProps> = ({ navigation }) => {
                         description={'Not a single thing'}
                         IconClass={ChartLineVariant}
                         actions={
-                            <Button mode={'outlined'} labelStyle={{ color: PXBColors.blue[500] }}>
+                            <Button mode={'outlined'} icon={'plus'}>
                                 Add a Device
                             </Button>
                         }
+                        iconColor={theme.dark ? PXBColors.black[200] : PXBColors.black[500]}
```

Comment:
Empty state is using `theme.colors.text` by default.

@huayunh are we okay with that or does it have to be black[200]? If it needs to be black[200] then we should put a @TODO here to come back and pull this out after we update the logic in the component.

Reply:
I personally prefer textSecondary but we can revisit this when we fix that blue font color thing Ian was complaining about

Reply:
Soooo... slap a @todo in here and leave the style as it is for now?